var/spack/repos/builtin/packages/boost/package.py

```diff
@@ -195,7 +182,7 @@ def determine_bootstrap_options(self, spec, withLibs, options):
         options.append("--with-libraries=%s" % ','.join(withLibs))
 
         if '+python' in spec:
-            options.append('--with-python=%s' % python_exe)
+            options.append('--with-python=%s' % spec['python'].executable)
```

Comment:
I'm getting an error here saying 'executable' is not defined.
I am using an external python here. Do these attributes get defined in that case?